add new telemetry events for features behind a SKU (#7543)
mattermost/docs: source/manage/telemetry.rst

  - *System Console Menu Discovery Diagnostics:* Clicks on the hamburger menu items of the System Console, including Administrator's Guide, Troubleshooting Forum, Commercial Support, About Mattermost, and clicks on the left-hand side navigation menu items
  - *In Product Notices Diagnostics:* Notices viewed, and the notices on which an action button was clicked.
  - *Threaded discussions:* Clicks to reply to a thread, reply using the footer element, filter threads by unread, mark as read, access to global threads section.

>   - *Custom Groups:* Invite people to a channel by using a custom group, mention a custom group, or modify a custom group.

Reviewer: ```suggestion
  - *Custom Groups:* Invite people to a channel by using a custom group, mention a custom group, and modify a custom group.
```

>   - *Read-Only Channels:* Navigate to a read-only channel, post a message to a read-only channel.

Reviewer: ```suggestion
  - *Read-Only Channels:* Navigate to a read-only channel, and post a message to a read-only channel.
```

  - *Shared Workspaces:* Navigate to a shared channel, post a message in a shared channel, and mention a remote user.

>   - *Guest Accounts:* Mention a guest account, DM a guest, add a guest to a channel.

Reviewer: ```suggestion
  - *Guest Accounts:* Mention a guest account, directly message a guest, and add a guest to a channel.
```
